zerver/lib/outgoing_webhook.py

```diff
@@ -27,18 +27,6 @@ def __init__(self, token: str, user_profile: UserProfile, service_name: str) ->
         self.user_profile = user_profile  # type: UserProfile
         self.service_name = service_name  # type: str
 
-    def build_bot_request(self, event: Dict[str, Any]) -> Optional[Any]:
-        raise NotImplementedError()
-
-    # Given a successful outgoing webhook REST operation, return
-    # a dictionary with `content` and other relevant fields.
-    # The main use case for this function is to massage data from
-    # various APIs to have similar data structures.
-    # It also allows bots to explictly set response_not_required.
-    def process_success(self, response_json: Dict[str, Any],
-                        event: Dict[str, Any]) -> Optional[Dict[str, Any]]:
-        raise NotImplementedError()
-
```

Comment:
I think the main reason to want these is for mypy type-checking, and that's only relevant if we're passing around `OutgoingWebhookServiceInterface` objects.